### PR TITLE
pd-configuration: fix the log level

### PR DESCRIPTION
**Pull request by nolouch**
Signed-off-by: nolouch <nolouch@gmail.com>

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- Other reference link(s): https://github.com/tikv/pd/pull/4519

### CODE REVIEW

File: pd-configuration-file.md [hunk 1]
```diff
@@ -123,8 +123,8 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
 
 ### `level`
 
-+ 日志等级，可指定为 "DEBUG"，"INFO"，"WARNING"，"ERROR"，"CRITICAL"。
-+ 默认值："INFO"
++ 指定日志的输出级别，可选项为 [debug, info, warn, error, fatal]。
```
Comment (en-jin19) [inline pd-configuration-file.md:126]:
```suggestion
+ 指定日志的输出级别，可指定为 "debug"，"info"，"warn"，"error"，" fatal"。
```

Comment (en-jin19) [inline pd-configuration-file.md:126]:
```suggestion
+ 指定日志的输出级别，可指定为 "debug"，"info"，"warn"，"error"，" fatal"。
```

**Review by en-jin19 — APPROVED** [review]
LGTM

**Comment by en-jin19** [comment]
@nolouch could you please invite a tech reviewer? thanks!

**Review by rleungx — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]
LGTM

File: pd-configuration-file.md [hunk 2]
```diff
@@ -123,12 +123,14 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
...
-+ 日志等级，可指定为 "DEBUG"，"INFO"，"WARNING"，"ERROR"，"CRITICAL"。
-+ 默认值："INFO"
++ 指定日志的输出级别。
++ 可选值："debug"，"info"，"warn"，"error"，"fatal"
++ 默认值："info"
 
 ### `format`
 
-+ 日志格式，可指定为"text"，"json"，"console"。
++ 日志格式
```
Comment (en-jin19) [inline pd-configuration-file.md:132]:
```suggestion
+ 日志格式。
```

**Comment by en-jin19** [comment]
待确认是否要修改 [PD 命令行参数文件中 `--log-file`](https://docs.pingcap.com/zh/tidb/stable/command-line-flags-for-pd-configuration/#--log-file)的后半部分内容。当 `max-day`默认值变为 `0` 后（PD 部分的该内容不是 5.4 的新修改)，不再带有后半句功能。

**Review by ran-huang — APPROVED** [review]

**Pull request merged**